### DISCUSSION TITLE
Choosing the right firmware for ESP32-DevKitC-32

### DISCUSSION THREAD

**Discussion by Laserhawk777** [MoonModules/WLED-MM#184] [Boards]
I want to flash an ESP32-DevKitC-32 and am not completely sure which firmware variant to choose.  Any help would be greatly appreciated.

**Comment by softhack007** [comment]
It looks like you have a normal esp32 with 4MB flash.
Please try one of the `esp32_4MB_` builds (see https://github.com/MoonModules/WLED/discussions/207#discussioncomment-11697620)

You can install WLED MoonModules with this web-based installer: https://wled-install.github.io


If you want more control over the details, use the adafruit web-based frontend for esptool https://adafruit.github.io/Adafruit_WebSerial_ESPTool
